kafka/consumer/group.py

```diff
@@ -527,6 +527,14 @@ def pause(self, *partitions):
             log.debug("Pausing partition %s", partition)
             self._subscription.pause(partition)
 
+    def paused(self):
+        """Get the partitions that were previously paused by a call to pause().
+
+        Returns:
+            set: {partition (TopicPartition), ...}
+        """
+        return self._subscription.paused_partitions()
```

Comment:
The java client returns a `Collections.unmodifiableSet` here. We could return a `frozenset`, but I stuck with `set` to be consistent with `assignment()` and `subscription()`.